frontend/src/store/modules/activity.ts

```diff
@@ -122,6 +122,7 @@ const actions = {
       "type=bb.sql-editor.query",
       `user=${currentUser.id}`,
       `limit=${limit}`,
+      `level=INFO`,
```

Comment:
let's add a comment to explain we only fetch the successful query.

It's not obvious that level=INFO indicates the successful query until looking at the backend implementation